lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -243,8 +243,8 @@
       data-add-button-label="${_("Add Community TA")}"
     ></div>
   %endif
-  
-  %if section_data['access']['instructor'] and settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
+
+  %if section_data['access']['instructor'] and settings.FEATURES.get('CUSTOM_COURSES_EDX', False) and section_data['ccx_is_enabled']:
```

Comment:
The check for `settings.FEATURES.get('CUSTOM_COURSES_EDX', False)` is unnecessary since you already include that in the logic to calculate `section_data['ccx_is_enabled']`